bootstrap-docker-install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if [ -f /etc/os-release ]; then
+    # freedesktop.org and systemd
+    . /etc/os-release
+    OS="${ID}-${VERSION_ID}"
+else
+    # We expect RHEL or Centos 7 for now. Lets set something as fallback
+    OS="$(uname -s)-$(uname -r)"
+fi
+
+echo "checking docker executable"
+if ! [ -x "$(command -v docker)" ]; then
+  case $OS in
+  rhel*)
+    extrasrepo=$(cat /etc/yum.repos.d/redhat* | grep -E '^\[.*extras.*\]$' | grep -vE 'debug|source' | tr -d '[|]')
+    sudo yum install -y --enablerepo $extrasrepo docker
+    sudo systemctl enable docker.service
+    sudo systemctl start docker.service
+    sudo setenforce 0
+    ;;
+  *)
+    echo "installing docker"
+    sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
```

Comment:
just to be safe? `which yum-config-manager || sudo yum install -y yum-utils`

Reply:
kk good point